executables/espresso/staticContext/average.x.yml

```diff
@@ -0,0 +1,11 @@
+average: &average
```

Comment:
Where is the documentation for all the work done in this PR? For example, what is what is `staticContext`, `common_control`????

We need to clarify the approach and put it in the README